guides/CorrectingTrackLabellingSteps.md

```diff
@@ -0,0 +1,187 @@
+  - Check the ID change took effect:
+    - You can go back to the usual one-frame visualisation by clicking on the grid icon (now crossed-out).
+    - See if the changes happened correctly.
+
+- If the crab has been re-ID for only a few frames: it may be easier to edit the ID in each frame (rather than following the grid method).
+
+  - Make sure the Toggle Annotation Editor is closed.
+  - Click on the bounding box whose ID we need to edit: a box with the track number will appear next to it.
+  - Simply edit the correct ID there.
+
+- If there is a false positive crab:
+  - Follow the steps above to edit the track ID, but instead delete it.
+  - If you cannot delete it (we saw this happen in Firefox), edit the track ID to be `_` (underscore).
+
+#### 4.2 Notes:
+
+- Notice that some crabs appear at later frames (not all crabs are present in frame 1), and their tracks should be checked too.
+- If there are false positives whose tracks are not very long-lived (under 6 frames), we will discard them, so don't worry about deleting them.
+- We saw some small crabs that are not detected. We wouldn't recommend to label and track these ones, as we don't think they were labelled in the detector dataset.
+
+> [!TIP]
+>
+> Browser:
+>
+> - We tried using the following:
+> - Edge: good results, so far the best behaving.
+> - Firefox: some issues with deleting track IDs.
+> - Chrome: some issues saving an edited track ID.
+> - Safari: temperamental, it will randomly reload the page and you will lose the unsaved work. Save regularly.
+
+> [!TIP]
+>
+> - Some convenient shortcuts for track correction:
+>   - Press `Up` arrow to toggle between the track ID of the bounding box and the bounding box number.
+
+> [!CAUTION]
+> There is no undo action in the annotation tool!
+>
+> - Be sure to save frequently, at least every labelled frame.
+> - The easiest way to save is as a VIA json file (click `Project` > `Save`) - this captures the full project.
+> - If you are using the "copy+paste" approach to reuse the annotations of the previous frame: note that the copied annotations stay in the clipboard after pasting!
+
+### 5. Save and reloading an unfinished project
+
+The easiest way to save a VIA project is by saving a VIA json file:
+
+- In the top bar, click `Project` > `Save`.
+- Leave all settings `ON` (as by default) and click `OK` to export.
+- This generates a json file with the name `<project_name>.json` that we can load again later in VIA to continue labelling.
+
+To later load and continue working on an unfinished project:
+
+- In the top bar, click `Project` > `Load`, and then select the VIA json file.
+
+### 6. Export the final annotations
+
+Please export the final annotations as a csv format:
+
+- in the top menu, click `Annotation` > `Export annotations (as csv)`
+- This will download a csv file to your Downloads folder named `<project_name>_csv.csv`
```

Comment:
Just confirming that you want "_csv.csv" (not just .csv, like .json)?

Reply:
Yeah it's a bit awkward but this is the convention of the VIA annotator. It will just download a file with that name (and no option to change it before, but I think that is alright).